src/YAMLValidator/Constants.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.YamlValidator;
+public class Constants
+{
+    public const string FileTypeName = "file";
+    public const string FolderTypeName = "folder";
+    public const string YamlFileExtension = ".yaml";
+    public const string YmlFileExtension = ".yml";
+    public const string JsonFileExtension = ".json";
+
+    public const string verbose = "verbose";
```

Comment:
Style nitpick: `public const` and `public static readonly` fields should start upper case:
```suggestion
    public const string Verbose = "verbose";
```

Reply:
fixed